packages/shared/src/components/buttons/ButtonV2.tsx

```diff
@@ -0,0 +1,119 @@
+import React, {
+  forwardRef,
+  HTMLAttributes,
+  ReactElement,
+  ReactNode,
+  Ref,
+} from 'react';
+import classNames from 'classnames';
+import { IconProps } from '../Icon';
+import { Loader } from '../Loader';
+import { combinedClicks } from '../../lib/click';
+
+import { ColorName as ButtonColor } from '../../styles/colors';
+import {
+  ButtonSize,
+  ButtonVariant,
+  ButtonIconPosition,
+  useGetIconWithSize,
+  IconOnlySizeToClassName,
+  SizeToClassName,
+  VariantColorToClassName,
+  VariantToClassName,
+} from './common';
+
+export type IconType = React.ReactElement<IconProps>;
+
+export { ButtonColor, ButtonSize, ButtonVariant, ButtonIconPosition };
```

Comment:
This feels a bit odd, is it only because we supporting both now and you want to be able to import from 1?
Meaning this should go away once we delete the old one?

Reply:
I wanted to be able to control the `ButtonV2` with only one import. If I did not export these fields here, we would end up with 2 imports - one for Button component itself and one for the common button types. I think this is clearer from the consumer perspective, the consumer does not need to care where the `ButtonSize` enum is defined.

Reply:
Yeah that was my understanding indeed.
So when we remove `Button` (v1) we can remove this as well correct?